views/components/GraphTable/helpers.js

```diff
@@ -24,12 +20,18 @@ const sortAllSitesBeforeTotalsSite = (siteA, siteB) => {
   if (siteB.name === TOTALS) return -1
 }
 
-export const formatSiteData = (studyCounts, studyTargets, studyPercentage) =>
-  !!studyTargets
-    ? `${studyCounts} / ${studyTargets} (${formatAsPercentage(
-        studyPercentage
-      )})`
-    : `${studyCounts} / (${formatAsPercentage(studyPercentage)})`
+export const formatGraphTableCellData = (studyCounts = 0, studyTargets) => {
```

Comment:
we call this `siteTarget` in the component. We should probably keep that consistent.

```suggestion
export const formatGraphTableCellData = (studyCounts = 0, siteTarget) => {
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/204/commits/c33a01801bbf0ce5a2bbc4b4e4ca72a59724be67